ruby-monday-blog/app/views/posts/_preview.html.erb

```diff
@@ -0,0 +1,22 @@
+<% hide_tags ||= false %>
+<hr />
+<%= div_for post, class: 'post-preview' do %>
+  <%= link_to post do %>
+    <h2><%= post.title %></h2>
+  <% end %>
+  <p class="post-timestamp">Posted <%= time_ago_in_words post.created_at %>&nbsp;ago</p>
+  <p class="post-body"><%= post.body %></p>
+  <ul class="list-inline clearfix">
+    <% unless hide_tags %>
+      <% post.tags.each do |tag| %>
+        <% unless tag.content.empty? %>
```

Comment:
This is a great use case for a Null Object / Default.  If there are no tags, then instead of `tags` returning essentially `Tag.none`, it can return `UncategorizedTag` or whatever PORO we specify